linorobot2_bringup/launch/depth.launch.py

```diff
@@ -61,5 +63,15 @@ def generate_launch_description():
                 'base_frame': 'camera_link'
             }.items()   
         ),
+        
+        IncludeLaunchDescription(
+            PythonLaunchDescriptionSource(PathJoinSubstitution(
+                [FindPackageShare('depthai_examples'), 'launch/include', 'zed_camera.launch.py']
+            )),
+            condition=IfCondition(PythonExpression(['"', LaunchConfiguration('sensor'), '" in "', str(oakd_sensors), '"'])),
+            launch_arguments={
+                'camera_model': LaunchConfiguration('sensor'),                
```

Comment:
I would prefer this 

        ```
IncludeLaunchDescription(
            PythonLaunchDescriptionSource(PathJoinSubstitution(
                [FindPackageShare('depthai_examples'), 'launch', 'stereo.launch.py']
            )),
            condition=IfCondition(PythonExpression(['"', LaunchConfiguration('sensor'), '" in "', str(oakd_sensors), '"'])),
            launch_arguments={
                'camera_model': PythonExpression(["'OAK-D' if 'oakd' in ", LaunchConfiguration('sensor'), " else 'OAK-D-LITE' if 'oakdlite' in ", LaunchConfiguration('sensor'), " else 'OAK-D-PRO' if 'oakdpro' in ", LaunchConfiguration('sensor')," else 'UNKNOWN'"]),                
            }.items()   
        ),
```

otherwise valuemapper will file if e.g. realsense is used. this broke launch for me.